src/Build/BackEnd/Components/RequestBuilder/IntrinsicTasks/ItemGroupLoggingHelper.cs

```diff
@@ -41,7 +41,9 @@ internal static class ItemGroupLoggingHelper
         /// to materialize the Message as that's a declaration assembly. We inject the logic
         /// here.
         /// </summary>
+#pragma warning disable CA1810 // Initialize reference type static fields inline
```

Comment:
Actually, I don't think this constructor is ever used; can we just delete it?

Reply:
I think it is a false positive. Since the variables that it is assigning to are not members of the class. But are static members of other static classes.

Moving those inline would change the logic.

Reply:
Gotcha. Sounds good.

Reply:
@elachlan can you please file an issue on https://github.com/dotnet/roslyn-analyzers about this false positive?